tox.ini

```diff
@@ -187,7 +187,7 @@ basepython = {[default]basepython}
 skip_install = True
 
 deps =
-    coverage==4.5.4  # rq.filter: <5  # coverage 5.0 drops Python 3.4 support
```

Comment:
rq.filter doesn't seem to be working

Reply:
You have to commit it to `master` before it takes effect.  Committing to this branch doesn't do anything.

Reply:
[It's on master](https://github.com/python-hyper/hyperlink/blob/master/tox.ini#L190)…

Reply:
…12 days ago. Lemme reverting that again.

Reply:
I can't figure out what we need to do to tell `requires.io` to stop updating this.  I must have the syntax wrong or something…  @glyph ?

Reply:
I don't see comments-in-the-comment in any other projects I've done with requires.io, so maybe rather than:

```
    coverage==4.5.4  # rq.filter: <5  # coverage 5.0 drops Python 3.4 support
```
do
```
    # coverage 5.0 drops Python 3.4 support
    coverage==4.5.4  # rq.filter: <5
```

And then close this PR to see what it does when it reopens it.